src/window-system/x11-cairo-surface.cpp

```diff
@@ -57,6 +57,14 @@ X11CairoSurface::X11CairoSurface(Display *display) : display(display) {
       CWColormap | CWBorderPixel | CWBackPixel | CWOverrideRedirect, &attr);
   XMapWindow(display, this->window);
 
+  // Add WM_NAME and WM_CLASS properties to the window
+  std::string name("touchégg");
```

Comment:
Yes, let's do it safe and avoid the "é", just in case some WM struggles with it:

```suggestion
  std::string name{"touchegg"};
```